### PR TITLE
Create inbox.graphql

### PR DESCRIPTION
**Pull request by ibrahimovich22**

### CODE REVIEW

**Comment by ibrahimovich22** [comment]
@esahmed21 got PN?

**Comment by ibrahimovich22** [comment]
@esahmed21 dddd

**Comment by ibrahimovich22** [comment]
@esahmed21 how now?

File: inbox.graphql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        isDone
+      }
+    }
+  }
+}
+
+
+mutation UnsubscribeFromNotification($id: ID!, $notificationId: ID!, $state: SubscriptionState!) {
+  updateSubscription(input: {subscribableId: $id, state: $state}) {
+    subscribable {
```
Comment (esahmed21) [inline inbox.graphql]: Wow

**Comment by esahmed21** [comment]
https://github.com/orgs/community/discussions/77264

**Pull request merged**